### DISCUSSION TITLE
get started

### DISCUSSION THREAD

**Discussion by LucaPiciullo** [MoiseRousseau/PyGeoStudio#3] [General]
Hello @MoiseRousseau what you have seems interesting. I was trying to test the code but I am having some issues to ge started. Can you please give more details about how to start? thanks

**Comment by MoiseRousseau** [comment]
Hi Luca,
What did you try ? Do you have an error message ? Did you successfully installed the library ? Which example code are you trying to run ?
Moise

**Reply by LucaPiciullo** [reply]
Hello Moise,

Thank you very much for your quick reply. I am very interesting to use you code for a research project that I am working on. 
I have installed the library and I was trying to navigate among the different codes in the folder. 
The PyGeoStudio code is running properly, but what the code is generating is less clear to me and how to change the input paramters?

**Reply by MoiseRousseau** [reply]
In fact, PyGeoStudio unpack a GeoStudio study, and read the xml file at the root of the extracted folder that contains every study component (geometry, material, mesh, ...). Note I build the library initially for visualizing SEEP results, so other capabilities are less tested, and everything lacks documentation I admit. I will have a look later today to try to better document each example. You can still modify input parameter as in `Rapid_drawdown/change_saturated_perm.py` and write back the modified study with `geofile.writeGeoStudioFile(out_study)`.

For information, what are you trying to do with pgs (PyGeoStudio) ?

**Reply by LucaPiciullo** [reply]
I would like to do 2 things: - run several simulations - quickly run with python a seep+slope analysis using as inputs the forecasted meterelogical data. The final aim is to have a real time slope stability analysis that would work as early warning. We would like to continue the work published here: [https://link.springer.com/article/10.1007/s11069-022-05524-3](https://link.springer.com/article/10.1007/s11069-022-05524-3)

**Reply by MoiseRousseau** [reply]
So you want to modify programmatically in the analysis the boundary condition of the model of the model in figure 5 of the article, and then run the analysis ? This sounds possible, but there are limitation with the current status of PyGeoStudio, as 1) Analysis cannot be run programmatically, and 2) Boundary conditions are not yet modifiable.
However, this could be implemented, and we can discuss on how we can collaborate on this. Please email me at rousseau [dot] moise @ gmail [dot] com

**Reply by MoiseRousseau** [reply]
Updated my email (lacked the dot)
